go-mode.el

```diff
@@ -1408,7 +1408,7 @@ It looks for archive files in /pkg/."
   (reverse (remove nil
                    (mapcar
                     (lambda (line)
-                      (when (string-match "^\\(.+\\):\\([[:digit:]]+\\): imported and not used: \".+\".*$" line)
+                      (when (string-match "^\\(.+?\\):\\([[:digit:]]+\\):\\([[:digit:]]+:\\)? imported and not used: \".+\".*$" line)
```

Comment:
Is there a specific reason you switched from `.+` to `.+?` for the file name? That shouldn't be necessary here as far as I am aware.